actions/assistant.py

```diff
@@ -169,13 +169,16 @@ def _install_repo(repo: Dict[str, str], remove_dir: bool = True) -> List[str]:
         if "checkout" in repo:
             assert isinstance(repo["checkout"], str)
             cmds.append(f"git checkout {repo['checkout']}")
+
+        if "requirement_file" in repo:
+            reqs = repo["requirement_file"]
+            reqs = [reqs] if isinstance(reqs, str) else reqs
+            cmds.append(f"pip install --quiet --upgrade {' '.join([f'-r {req}' for req in reqs])}")
```

Comment:
As a user, its plural sounds more natural IMO.
```suggestion
        if "requirement_files" in repo:
            reqs = repo["requirement_files"]
            reqs = [reqs] if isinstance(reqs, str) else reqs
            cmds.append(f"pip install --quiet --upgrade {' '.join([f'-r {req}' for req in reqs])}")
```